_topic_map.yml

```diff
@@ -265,23 +265,25 @@ Name: Installing Clusters
 Dir: install
 Distros: openshift-origin,openshift-enterprise
 Topics:
-- Name: Planning Your Installation
+- Name: Planning your installation
   File: index
-- Name: System and Environment Requirements
+- Name: System and environment requirements
   File: prerequisites
-- Name: Preparing Your Hosts
+- Name: Preparing your hosts
   File: host_preparation
 - Name: Configuring Your Inventory File
   File: configuring_inventory_file
 - Name: Example Inventory Files
```

Comment:
Will you PTAL at the rendered version of the Example Inventory Files topic? I might just be confused, but I am not sure that all the scenarios are valid with etcd on static pods.

Reply:
@gpei PTAL

Reply:
About the `Example Inventory Files` topic

- etcd related inventory file configuration lgtm. We still support both staic pod etcd and rpm etcd in ocp-3.11. When etcd co-located with master, then it would run as static pod on master node, or else etcd would be installed as stand-alone rpm etcd. One thing we could mention here is etcd stand-alone hosts on atomic is no longer supported. 

- Some outdated parameter in the example inventory file "Multiple Masters Using HAProxy Inventory File"
    - openshift_node_kubelet_args is deprecated, could use node config edits instead 
    - openshift_clock_enabled=true is redundant since all the places calling it were set to True by default
```
# apply updated node defaults
openshift_node_kubelet_args={'pods-per-core': ['10'], 'max-pods': ['250'], 'image-gc-high-threshold': ['90'], 'image-gc-low-threshold': ['80']}

# enable ntp on masters to ensure proper failover
openshift_clock_enabled=true
```

Reply:
@gpei, if users wanted to use static etcd pods on masters, what would they need to specify in the inventory file?

Is the static pod the preferred etcd deployment method in 3.10 and 3.11?

When was `openshift_node_kubelet_args` deprecated? I see several references in the collection.

Reply:
>When was openshift_node_kubelet_args deprecated? I see several references in the collection.

In 3.10, during the move to node group configmaps. Instead the node edits should be used:
```
openshift_node_groups=[{'name': 'node-config-all-in-one', 'labels': ['node-role.kubernetes.io/master=true', 'node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true'], 'edits': [{ 'key': 'kubeletArguments.pods-per-core','value': ['20']}]}]
```

Reply:
I'm making an issue to track any other openshift_node_kubelet_args deprecation changes that aren't included in this PR: https://github.com/openshift/openshift-docs/issues/12200